### DISCUSSION TITLE
tooltip

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1103] [Q&A] [answered]
how show tooltip in browser by mobile devices (iphone,ipad ...) ?

![image](https://user-images.githubusercontent.com/18022720/113386908-cfd6e080-93bd-11eb-9da0-d921579dae95.png)

**Comment by bbastings** [comment, marked as answer]
Currently you will only get the locate tooltip when using touch input with tools that enable snapping (AccuSnap.enableSnap). So tools like Measure Location or Measure Distance that enable the virtual mouse cursor. In an upcoming release we will also add a ToolSettings option to enable the virtual mouse cursor for locate. So as of now this is an example for mouse input only.
